airbyte-integrations/connectors/source-shopify/pyproject.toml

```diff
@@ -3,7 +3,7 @@ requires = [ "poetry-core>=1.0.0",]
 build-backend = "poetry.core.masonry.api"
 
 [tool.poetry]
-version = "2.1.0"
+version = "2.1.2"
```

Comment:
Teeny nit; should this be set to 2.1.1?

Reply:
Nope, this is the next one after this one: https://github.com/airbytehq/airbyte/pull/38352